readme.md

```diff
@@ -31,6 +33,22 @@ As a computer expert, we want to be more efficient and do our jobs better. We kn
 1. [remove a folder](#removefolder)
 1. [list folder contents](#listfolder)
 
+## <a id="findFile"></a>find a stale file
+
+Find all files modified more than 5 days ago
+
+```bash
+find . -mtime +5
+```
+
+## <a id="showCal"></a>show a calendar
+
+Display a text calendar, along with the week numbers
+
+```
+cal -w
```

Comment:
Hmmm, I'm installing debian right now to test, maybe I'll change it to display three calendars, this month and prev/next month:
```
cal -3
```